### PR TITLE
fix: renamed protected parameter properties

### PR DESCRIPTION
**Pull request by BeksOmega**
## The basics

- [ ] I branched from develop
- [ ] My pull request is against develop
- [X] My code follows the [style guide](https://developers.google.com/blockly/guides/modify/web/style-guide)
- [X] I ran `npm run format` and `npm run lint`

### Resolves

N/A

### Proposed Changes

The automatic migration script renamed some properties when transforming them into parameter properties. This is fine for private properties, but protected properties are part of our public API, and constitute a breaking change when they are renamed.

#### Behavior Before Change

Properties were renamed, creating a breaking change.

#### Behavior After Change

Property renames are reverted.

### Reason for Changes

Lessen the number of accidental breaking changes we're committing.

### Test Coverage

No manual testing needed.

### Documentation

N/A

### Additional Information

Also going to do a sweep for public properties that might have gotten renamed.

### CODE REVIEW

File: core/gesture.ts [hunk 1]
```diff
@@ -172,13 +172,18 @@ export class Gesture {
...
+  private mostRecentEvent_: Event;
+
   /**
    * @param e The event that kicked off this gesture.
    * @param creatorWorkspace The workspace that created this gesture and has a
    *     reference to it.
    */
   constructor(
       private e: Event, private readonly creatorWorkspace: WorkspaceSvg) {
+    this.mostRecentEvent_ = e;
```
Comment (rachel-fenichel) [inline core/gesture.ts:185]: Why was this not set before?

Comment (BeksOmega) [reply core/gesture.ts:185]: The automatic migration created a parameter property, and renamed everything to `this.e`.

With your comment, I see I forgot to remove the `private` from `private e: Event`. Will check for similar misses & fix.

**Review by rachel-fenichel — APPROVED** [review]

**Comment by BeksOmega** [comment]
Realized I forgot to do a test build on this branch. Did a build, and found I had errors because I forgot to revert the renames in subclasses. Would love if you could take another quick look before I merge @rachel-fenichel

**Review by rachel-fenichel — APPROVED** [review]
Subclasses! *shakes fist*

File: core/block_dragger.ts [hunk 2]
```diff
@@ -60,17 +65,21 @@ export class BlockDragger implements IBlockDragger {
    * @param block The block to drag.
    * @param workspace The workspace to drag on.
    */
-  constructor(
-      private readonly block: BlockSvg,
-      private readonly workspace: WorkspaceSvg) {
+  constructor(block: BlockSvg, workspace: WorkspaceSvg) {
```
[the remark this answers is not in the document]
Comment (BeksOmega) [reply core/block_dragger.ts:68]: Yeah we could have definitely done it that way, but I wanted to include the old doc strings for the protected properties, without messing up the doc strings for the constructor parameters.

It's also useful to have the properties be separate if we want to deprecate these protected properties, and create new ones that conform to the new styleguide. That way we can apply an `@deprecated` tag at least.

[the remark this answers is not in the document]
Comment (cpcallen) [reply core/block_dragger.ts:68]: I figured you'd had a good reason—and indeed you did.

**Pull request merged**